RubeesEat/Pages/ShowBills.cshtml

```diff
@@ -5,57 +5,67 @@
+        if (!Int32.TryParse(Request.Query["page"], out page) || page <= 0)
+        {
+            page = 1;
+        }
+
+        var billsPaginated = billRepository.GetRecentBalanceChanges(currentUser, page, pageSize);
+    }
+
+    <ul class="entryLines list-group mt-5 mx-auto" style="max-width: 900px;">
+        @foreach (var item in billsPaginated.Items)
+        {
+        <li t-id="entryLines > li" class="list-group-item justify-content-between">
+            <a class="text-decoration-none text-dark w-100" t-action="showBillDetails" href="BillDetails/@item.BillId">
                 <div class="amount-description">
-                    <span class="amount @(item.Amount < 0 ? "negative-amount" : "positive-amount")" t-id="amount">
-                        @item.Amount€
-                    </span>
-                    <span class="description" t-id="description">
-                        @item.Description
-                    </span>
-                </div>
-                <div class="date" t-id="date">
-                    @item.Date.ToString("D", CultureInfo.CreateSpecificCulture("en-US"))
+                    <div class="row">
+                        <div class="col-3 @(item.Amount < 0 ? "text-danger" : "text-success")" t-id="amount">
+                            @(item.Amount)€
+                        </div>
+                        <div class="col-5" t-id="description">
+                            @item.Description
+                        </div>
+                        <div class="col-4 text-end fst-italic" t-id="date">
+                            @item.Date.ToString("D", CultureInfo.CreateSpecificCulture("en-US"))
+                        </div>
+                    </div>
                 </div>
             </a>
         </li>
-    }
-</ul>
+        }
+    </ul>
 
-<div class="pagination">
-    @if (billsPaginated.HasPreviousPage)
-    {
-        <a href="../ShowBills/?page=@(page - 1)" t-action="prevButton" t-click-behavior="Click">Prev</a>
-    }
+    <div class="pagination">
+        @if (billsPaginated.HasPreviousPage)
+        {
+            <a href="../ShowBills/?page=@(page - 1)" t-action="prevButton" t-click-behavior="Click">Prev</a>
+        }
 
-    <span class="pageInfo">Viewing page @(page)/@billsPaginated.TotalPages</span>
+        <span class="pageInfo">Viewing page @(page)/@billsPaginated.TotalPages</span>
 
-    @if (billsPaginated.HasNextPage)
+        @if (billsPaginated.HasNextPage)
+        {
+            <a href="../ShowBills/?page=@(page + 1)" t-action="nextButton" t-click-behavior="Click">Next</a>
+        }
+    </div>
+
+    @if (!billsPaginated.Items.Any())
     {
-        <a href="../ShowBills/?page=@(page + 1)" t-action="nextButton" t-click-behavior="Click">Next</a>
+        <div class="errorMessage">No bills found for this page.</div>
     }
-</div>
-
-@if (!billsPaginated.Items.Any())
-{
-    <div class="errorMessage">No bills found for this page.</div>
-}
\ No newline at end of file
+</div>
```

Comment:
It looks a bit better if the pagination is centered and has a bit of a gap between the list